test/unit/attributes.js

```diff
@@ -451,7 +451,7 @@ QUnit.test( "attr(String, Object)", function( assert ) {
 
 	$radio = jQuery( "<input>", {
 		"value": "sup",
-		"type": "radio"
+		"TYPE": "radio"
```

Comment:
To make sure we use the type attrhook even if the name that comes in is not lowercase.